src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,12 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
+
+        @JvmStatic
+        fun removeLetter(input: String) : CharArray {
+            val replaced = input
+                .lowercase()
+                .replace('r','!')
```

Comment:
Nice!

You also may write this method via single line for visual simplification.
Smth like this
 fun removeLetter(input: String) = input.lowercase().replace('r','!').toCharArray()